src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -100,17 +100,26 @@ protected function getFillsSectionContent()
      */
     protected function getBordersSectionContent()
     {
-        return <<<EOD
-<borders count="1">
-    <border>
-        <left/>
-        <right/>
-        <top/>
-        <bottom/>
-        <diagonal/>
-    </border>
-</borders>
-EOD;
+        $content = '<borders count="' . count($this->styleIdToStyleMappingTable) . '">';
```

Comment:
instead of using `$this->styleIdToStyleMappingTable`, can you use `$this->getRegisteredStyles()`? It's more explicit.

```php
$registeredStyles = $this->getRegisteredStyles();
$registeredStylesCount = count($registeredStyles);

$content = '<borders count="' . $registeredStylesCount . '">';

foreach ($registeredStyles as $style) {
...
```